core/trino-main/src/main/java/io/trino/operator/project/InputChannels.java

```diff
@@ -25,15 +26,23 @@
 public class InputChannels
 {
     private final int[] inputChannels;
+    private final int[] eagerlyLoadedChannelIndexes;
```

Comment:
Are these indexes into the `inputChannels` array or the channel numbers of the channels to load eagerly?

Reply:
This was indexes into inputChannels array earlier, but i've changed it now to be channel numbers of the channels to load eagerly since this seems easier to follow.